src/annotations/api/index.test.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+
+import {mocked} from 'ts-jest/utils'
+import {writeAnnotation} from 'src/annotations/api'
+import {API_BASE_PATH} from 'src/shared/constants'
+import axios from 'axios'
+
+jest.mock('axios')
+
+describe('annotations api calls', () => {
+  describe('POST - write annotations api calls', () => {
+    it('can write an annotation successfully and return the array of annotations', async () => {
+      const data = [
+        {
+          summary: 'GO PACK GO',
+          'start-time': 'beginning of time',
+          'end-time': Date.now(),
+          stream: 'Lambeau Field',
+        },
+      ]
+      mocked(axios.post).mockImplementationOnce(() =>
+        Promise.resolve({status: 200, data})
+      )
+
+      const response = await writeAnnotation([{summary: 'Hey there'}])
+
+      expect(axios.post).toHaveBeenCalledTimes(1)
+
+      expect(response).toEqual(data)
+    })
+    it('makes a post request with annotations in the body of the request', async () => {
```

Comment:
this test can be removed. it's testing implementation that will change when we remove axios